root/static/scripts/common/utility/formatDate.js

```diff
@@ -11,14 +11,14 @@ import ko from 'knockout';
 
 import {fixedWidthInteger} from './strings.js';
 
-function formatDate(date: PartialDateT | null): string {
+function formatDate(date?: PartialDateT | null): string {
```

Comment:
Last time I checked, Flow doesn't correctly interpret `date?:` to mean that the argument can be omitted but can't be undefined (it allows passing undefined here), unlike the object type suggestion I made, but I'd still do the same as there for clarity, assuming the intent is the same. :)

```js
date: ?PartialDateT
```

Reply:
I think in this case the intention is clearly to allow it being set as `undefined` rather than nothing at all being passed to `formatDate` so I made the change. The *partial date* sections though have the same issue as on my comment above.